src/game/gui/menu.c

```diff
@@ -184,9 +188,11 @@ static int menu_action(component *mc, int action) {
             c = sizer_get(mc, m->selected);
 
         } while(component_is_disabled(c));
-        // Play menu sound
-        audio_play_sound(19, 0.5f, 0.0f, 2.0f);
-        component_select(c, 1);
+        if(c != old_c) {
+            // Play menu sound
+            audio_play_sound(19, 0.5f, 0.0f, 2.0f);
```

Comment:
If you feel like it, there could be either
a) Constants for the default values here, or
b) audio_play_sound_with_defaults(19) or somesuch

This is not necessarily part of this PR, just came to mind while reading.